rootfs/Dockerfile

```diff
@@ -1,7 +1,5 @@
-FROM alpine:3.3
-
-RUN apk -U add ca-certificates && \
-    rm -Rf /var/cache/apk/*
+FROM gcr.io/google_containers/ubuntu-slim:0.2
 COPY ./bin/boot /bin/boot
+RUN apt-get update && apt-get -y install ca-certificates && apt-get clean
```

Comment:
https://github.com/kubernetes/contrib/blob/master/git-sync/Dockerfile#L6-L9 you will probably want to add in the last `rm` call